addons/assigngear/functions/fn_onEdenMessageRecieved.sqf

```diff
@@ -33,6 +33,10 @@ if (_messageId isEqualTo 0) then { //Mission saved.
         private _newdescription = preprocessFile "description.ext";
         if (!(_newdescription isEqualTo GVAR(descriptionExt))) then {
 
+            // Clear all cached loadouts
+            {
+                GVAR(namespace) setVariable [_x, nil]
+            } forEach ((allVariables GVAR(namespace)) select {_x find "loadout" == 0});
```

Comment:
In theory it's probably quicker to just point GVAR(namespace) to a new namespace. Aka replacing this with something like (psudeo code here not SQF)
`GVAR(namespace) = CREATE_NEW_NAMESPACE`

SQF should then auto garbage collect